libs/utils/android/benchmark.py

```diff
@@ -47,7 +47,48 @@ class LisaBenchmark(object):
     directory.
     """
 
-    bm_conf = None
+    bm_conf = {
```

Comment:
Yes, I think this kind of make sense given the current code.
Moreover, a sub-class can also easily just re-define a part of the default configuration, for example to change the emeter to use... +1